nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -284,6 +284,14 @@ let
         ${buildMenuAdditionalParamsGrub2 config "video=1920x1080@60"}
       }
 
+      # If we boot into a graphical environment where X is autorunned
+      # and always crashes, it makes the media unusable. Allow the user
+      # to disable this.
+      submenu "Disable display-manager" --class quirk-disable-displaymanager {
```

Comment:
We can introduce an icon, if desired, and if one can be composed. They are composed from the [breeze icon theme](https://github.com/KDE/breeze-icons). If you see something in it that can be adapted, or used as an inspiration I can look into [adding it to the relevant artwork](https://github.com/NixOS/nixos-artwork/tree/master/bootloader/grub2-installer).

Reply:
I found breeze/devices/64/video-display.svg the most suitable.
I mostly wanted something like just a blank monitor, or communicates a tty.

Reply:
I'm highly partial to elementary's icons which appears to be exactly what I'm looking for

![Screenshot from 2019-09-28 21 14 48](https://user-images.githubusercontent.com/28888242/65824326-1e3d7900-e235-11e9-9c3f-8b7563553b88.png)

Reply:
Oh, forgot to specify the symbolic icons. Though the one from breeze, once made into outlines, may work, I'll try to remember to look into it later. (The PR can be merged without that change.)

Reply:
Ahh, there's computer/video-display symbolic icons that should be fine then.
I'll add an issue at nixos-artwork for that once merged @samueldr.